ProteoformSuiteGUI/ProteoformSweet.cs

```diff
@@ -347,33 +366,38 @@ private void saveMethod(string method_filename)
 
         private void loadRunToolStripMenuItem_Click(object sender, EventArgs e)
         {
-            var result = MessageBox.Show("Choose a method file.", "Method Load and Run", MessageBoxButtons.OKCancel);
-            if (result == DialogResult.Cancel) return;
-            if (Lollipop.input_files.Count != 0)
+            if (Lollipop.input_files.Count == 0)
             {
-                var response = MessageBox.Show("Would you like to use the files specified in LoadDeconvolution rather than those referenced in the method file?", "Multiple Deconvolution File References", MessageBoxButtons.YesNoCancel);
-                if (response == DialogResult.Yes) { Lollipop.use_method_files = false; }
-                if (response == DialogResult.No) { Lollipop.input_files.Clear(); Lollipop.use_method_files = true; }
-                if (response == DialogResult.Cancel) { return; }
+                MessageBox.Show("Please load in deconvolution result files in order to use load and run.");
```

Comment:
I think this is fine. It's unlikely we'd specify the files in the method file. Rather, we'd probably just use Save All and Open if we had already started a run.